src/Features/Core/Portable/ImplementAbstractClass/AbstractImplementAbstractClassService.Editor.cs

```diff
@@ -153,13 +153,7 @@ private partial class Editor
                     setMethod: setMethod);
             }
 
-            private bool ShouldGenerateAccessor(IMethodSymbol method)
-            {
-                return
-                    method != null &&
-                    method.IsAccessibleWithin(_state.ClassType) &&
-                    _state.ClassType.FindImplementationForAbstractMember(method) == null;
```

Comment:
Don't you want to keep this condition?  You don't want to re-implemenent already implemented members.  (Also, why didn't this fail a test?)